indico/modules/events/registration/controllers/management/reglists.py

```diff
@@ -600,8 +600,20 @@ def _process(self):
 class RHRegistrationHide(RHManageRegistrationBase):
     """Hide a registration from the participants lists."""
 
+    def _log_changes(self):
+        log_data = {'Email': self.registration.email}
```

Comment:
the friendly_id would be useful as well (either in the data or maybe even in the title of the log message)